### ISSUE TITLE
Search in Control Panel still big after install

### ISSUE THREAD

**Issue by Kradek200** [krlvm/ClassicSearch#4]
I dont have much else to say, here's a screenshot :

![image](https://user-images.githubusercontent.com/108480245/207630006-c61d01a4-e030-4812-b300-c04a1f98e8d8.png)

**Comment by krlvm** [comment]
Yes, I know it, unfortunately it is also present in some apps Open/Save dialogs.
I'm not sure if it is possible to do something with it at the moment.

**Comment by Kradek200** [comment]
> Yes, I know it, unfortunately it is also present in some apps Open/Save dialogs. I'm not sure if it is possible to do something with it at the moment.

Ah, no problem, i'll be patient until you maybe find a solution :)
